### ISSUE TITLE
Better descriptive labels for talk experience and talk preparation assistance

### ISSUE THREAD

**Issue by shiroyuki** [pyconca/2016-symposion#2]
![screen shot 2016-07-27 at 3 56 45 pm](https://cloud.githubusercontent.com/assets/226612/17190274/49bd23c2-5413-11e6-9715-16ce1519a8f2.png)

If we can change, we should change:

| Current Label | Proposed Label |
| :-- | :-- |
| “Speaker mentorship” | “Required assistance in talk preparation?” |

The hints should be updated according to the new label.

**Comment by myles** [comment]
I think _Previously given_ is asking if you have given this talk before at any conferences.

**Comment by shiroyuki** [comment]
You are right about **Previously given** that but it is not obvious enough that the user understands without reading the hint.

I think we should make it obvious what the question is about. We can either rephrase the label or change the label of the responses from "Yes" and "No" to "Yes, I have previously given talks before." and "No, I'm a first-time speaker." respectively.

**Comment by shiroyuki** [comment]
@myles Sorry... I misunderstood it. :(
